src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -77,50 +78,104 @@ class WorkspaceNewRoomPage extends React.Component {
     }
 
     /**
-     * Called when a workspace is selected. Also calls checkAndModifyRoomName,
-     * which displays an error if the given roomName exists on the newly selected workspace.
-     * @param {String} policyID
+     * Called when the "Create Room" button is pressed.
```

Comment:
```suggestion
     * Create the room when the "Create Room" button is pressed.
```

Reply:
@marcaaron  I will remove entire comment for this method `onSubmit()` as per the jsdocs section on `STYLE.md`. And, what about naming-convention should I change the name from `onSubmit()` to `submit()` only?

Reply:
`validateAndCreatePolicyRoom()` is suggested in next comment.